src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
```

Comment:
you can modify these methods to work with `this.result` and a single parameter, as `operate` should handle updating `result`. Like this `add(number) {this.result += number;}`.